Cpp/source/DrawingEngine/DrawingEngine.hpp

```diff
@@ -2,15 +2,19 @@
 
 #include <iostream>
 #include <stdint.h>
-#include "FrameBuffer/FrameBuffer.hpp"
-
+#include "../FrameBuffer/FrameBuffer.hpp"
+#include "../Drawable/Drawable.hpp"
+#include <memory>
+ 
 class DrawingEngine
 {
 protected:
     FrameBuffer* FrameBuff;
+    std::vector<std::unique_ptr<Drawable>> engine_drawables{};
```

Comment:
i'd say std::vector<std::shared_ptr< comst Drawable > > this way whoever adds the drawables to drawable is sure it won't be modified.

Reply:
still applies